.github/workflows/build-wasm.yml

```diff
@@ -23,12 +23,6 @@ jobs:
           target: wasm32-unknown-unknown
           default: true
 
-      - name: Build the datamodel crate with default features
-        uses: actions-rs/cargo@v1
-        with:
-          command: build
-          args: -p datamodel --release --target=wasm32-unknown-unknown
```

Comment:
I strongly suspect this is a no-op, and we test this more thoroughly through prisma-fmt-wasm already.